common/lib/xmodule/xmodule/course_module.py

```diff
@@ -774,6 +774,30 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    learning_info = List(
+        display_name=_("Course Learning Information"),
+        help=_("Specify what student can learn from the course."),
+        default=[''],
```

Comment:
Yes, Initially, I am rendering the learning info section with an empty fields. If it is not the requirement then I can set it to `[]`. Kindly let me know.